### ISSUE TITLE
Add support for pico registration 

### ISSUE THREAD

**Issue by windley** [Picolab/wrangler#4]
- explicit registration functions in a registrar rule
- a registrar ruleset that can be installed in a special registrar pico
- support different hosts (might require reimplementing event:send())

A more complex example of this idea is to use a registrar. A registrar is a third party that manages the registry on behalf of the instances. The registrar watches for <code>child_created</code> or <code>child_destroyed</code> events to know when instances are created or destroyed and registers or deregisters them as appropriate. The registrar also periodically checks the health of instances and automatically deregisters those it deems incapacitated. The registrar decouples instances from the registry. They can be oblivious to the existence of the registry.

**Comment by burdettadam** [comment]
Is this the same idea as a "fleet manager"? Just functionality provided with the implementation hidden from users?

**Comment by windley** [comment]
No, the fleet manager was a collection. It did serve this purpose but in a very specialized way. This would be more generalized. I'm going to put it on the backlog for now, but we don't want to lose track of the idea. I think we'll need it for spimes.
